index.html

```diff
@@ -59,23 +73,26 @@ <h1 class="work-main-heading">My Recent Works</h1>
         <hr class="divider" />
         <div class="main-card">
           <div class="post-image"></div>
-          <h4 class="work-title-post">Multi-Post Stores</h4>
-          <p class="work-supporting-text">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard dummy text ever
-            since the 1500s, when an unknown printer took a standard dummy text.
-          </p>
-          <ul class="languages-list">
-            <li>css</li>
-            <li>html</li>
-            <li>bootstrap</li>
-            <li>Ruby</li>
-          </ul>
-          <div class="btn-left">
-            <button type="button" class="btn-primary">See Project</button>
+          <div class="main-card-content">
+            <h4 class="work-title-post">Multi-Post Stores</h4>
+            <p class="work-supporting-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul class="languages-list">
+              <li>css</li>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <div class="btn-left">
+              <button type="button" class="btn-primary">See Project</button>
+            </div>
           </div>
         </div>
-        <div class="card">
+        <div class="card card-1">
           <h1 class="card-heading">Professional Art Printing Data</h1>
           <p class="card-text">
             A daily selection of privately personalized reads; no accounts or
```

Comment:
- [ ] In the Figma template, the hover behavior of the regular project card is to make the title, description, tags, and opacity disappear and make the `See Project` button appears at the bottom of the card(look at the image below). Kindly try to implement that functionality so the user can have a view of the image under the information before clicking on the see project button.

Figma design:
![image](https://user-images.githubusercontent.com/61919617/178039701-8ea2b2bc-44f7-411c-b96a-eeac314940e9.png)

Reply:
Hi sir, I think it is not possible without js and I am just using CSS and only CSS it is not possible how can I implement this functionality?